app/style/app.css

```diff
@@ -57,13 +57,18 @@ gaia-header #upload-link {
 }
 
 .install-list .install-button {
-  float: right;
-  color: rgba(0, 0, 0, 0.53);
-  font-size: 15px;
-  min-width: 40px !important;
-  border-radius: 20px;
-  font-style: italic;
-  padding: 5px 8px;
+  --button-background: var(--color-blue);
+  --button-background-active: var(--background);
+  --button-font-size: 12px;
+  --button-height: 30px;
+  min-width: 75px !important;
```

Comment:
We should figure out why this is needed. The button should ideally be sized such that it wraps the contents neatly. Right now, we're basically abusing ```min-width``` as an override for ```width```.